common/unified/matrix/dense_kernels.cpp

```diff
@@ -181,6 +260,33 @@ void sub_scaled(std::shared_ptr<const DefaultExecutor> exec,
 GKO_INSTANTIATE_FOR_EACH_VALUE_TYPE(GKO_DECLARE_DENSE_SUB_SCALED_KERNEL);
 
 
+template <typename ValueType>
+void sub_scaled_real_complex(std::shared_ptr<const DefaultExecutor> exec,
+                             const matrix::Dense<ValueType> *alpha,
+                             const matrix::Dense<std::complex<ValueType>> *x,
```

Comment:
or it is combined with the previous one by two templated ValueType

Reply:
Yes, that would be the alternative option. I wanted to avoid having to manually unroll the instantiation macros, so I went with this solution.

Reply:
I would also favor a single implementation with two templates, since the new kernels are completely identical to the old ones. I'm guessing that adding a new instantiation macro for that wouldn't be too bad, but that might be wrong.

Reply:
Agreed, I moved to a single kernel for both.